backend/src/nodes/utils/auto_split.py

```diff
@@ -12,6 +12,34 @@ class Split:
     pass
 
 
+def estimate_tile_size(
+    free: int,
+    total: int,
+    model_size: int,
+    img: np.ndarray,
+    img_element_size: int = 4,
+    allowed_usage: float = 0.8,
+) -> Union[int, None]:
+    h, w, c = get_h_w_c(img)
+    img_bytes = h * w * c * img_element_size
+    mem_required_estimation = (model_size / (1024 * 52)) * img_bytes
```

Comment:
This factor is very specific to PyTorch. Is that even applicable to NCNN?

Reply:
I haven't really tested it enough to know for sure, but it at least seems to work the same for ncnn